samples/client/petstore/csharp/OpenAPIClientNetCoreProject/src/Org.OpenAPITools/Org.OpenAPITools.csproj

```diff
@@ -20,9 +20,9 @@
   <ItemGroup>
     <PackageReference Include="FubarCoder.RestSharp.Portable.Core" Version="4.0.7" />
     <PackageReference Include="FubarCoder.RestSharp.Portable.HttpClient" Version="4.0.7" />
-    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
-    <PackageReference Include="JsonSubTypes" Version="1.6.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="JsonSubTypes" Version="1.2.0" />
```

Comment:
I think for this one, we'll need to keep the change to use 12.x and 1.6.0 for the dependencies.

Reply:
I'l do it with a separate PR.